lib/game.rb

```diff
@@ -0,0 +1,88 @@
+require 'pry'
+class Game
+  attr_accessor :board, :player_1, :player_2, :WIN_COMBINATIONS
+
+  WIN_COMBINATIONS = [
+    [0,1,2],
+    [3,4,5],
+    [6,7,8],
+    [0,3,6],
+    [1,4,7],
+    [2,5,8],
+    [0,4,8],
+    [2,4,6]
+  ]
+
+  def initialize(player_1=Players::Human.new("X"),player_2=Players::Human.new("O"),board=Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+  def current_player
+    @count = board.turn_count
+    if @count % 2 == 0
+      return player_1
+    else
+      return player_2
+    end
+  end
+
+  def won?
+    WIN_COMBINATIONS.each do |combo|
+      if @board.cells[combo[0]]=="X"&&@board.cells[combo[1]]=="X"&&@board.cells[combo[2]]=="X"
+      return combo
+    elsif @board.cells[combo[0]]=="O"&&@board.cells[combo[1]]=="O"&&@board.cells[combo[2]]=="O"
+      return combo
+    end
+  end
+    return false
+  end
+
+  def draw?
+    board.full? == true && won? == false
+  end
+
+  def winner
```

Comment:
Instead of calling `Game#won?` twice, consider using an if statement assignment. For example:

```ruby
if won = won?
  board.cells[won.first]
end
```

If `won?` returns true, then and only then will the local variable `won` be assigned, otherwise, the method will just abort with a false return value. But if `won` is assigned, we can use that variable instead of calling `won?` again within the if statement.